tests/test-dirs/type-enclosing/jane-street.t/run.t

```diff
@@ -63,8 +63,8 @@ escape characters in string literals, so we use the revert-newlines script.
   
   type t5 : value
       ^
-  With verbosity 0: "type t5 : value"
-  With verbosity 1: "type t5 : value"
+  With verbosity 0: "type t5"
+  With verbosity 1: "type t5"
```

Comment:
There continues to be a diff in this test. `type t5 : value` should be printed here, as that's what the user wrote. Previously, the code checked the `jkind_annotation` in the typedtree to determine that.